<robot name="ur3e">
  <gripper name="gripper" clearance="0.08">
    <position xyz="0 0 0" xyzw="0.5  0.5  0.5 -0.5"/>
    <link name="collision_cylinder"/>
</gripper>

  <disable_collisions link1="shoulder_link"
     link2="upper_arm_link" reason=""/>
  <disable_collisions link1="upper_arm_link"
		      link2="forearm_link" reason=""/>

  <disable_collisions link1="wrist_1_link"
		      link2="wrist_2_link" reason=""/>
  <disable_collisions link1="wrist_2_link"
		      link2="wrist_3_link" reason=""/>
  <disable_collisions link1="shoulder_link"
		      link2="forearm_link" reason=""/>
  <disable_collisions link1="wrist_1_link"
		      link2="wrist_3_link" reason=""/>
  <disable_collisions link1="base_link_inertia"
		      link2="shoulder_link" reason=""/>
  <disable_collisions link1="forearm_link"
		      link2="wrist_1_link" reason=""/>

</robot>
